Christopher-christpheranderson/homework-1/fizzbuzz/js/main.js

```diff
@@ -0,0 +1,12 @@
+for (i = 1; i < 100; i++){
+	if(i % 3){ 
```

Comment:
Adding a space after keywords like `if` will make your code more readable for humans.